src/runner/browser-env/vite/browser-modules/@types/vite.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module "virtual:@testplane/driver" {
```

Comment:
declare [virtual vite module](https://vitejs.dev/guide/api-plugin.html#virtual-modules-convention). Used in order to pass `automationProtocolPath` from server to client.